helpers.js

```diff
@@ -99,6 +110,13 @@ let getFiles = (path) => {
     return glob.sync("**/*.js", {'ignore': ['node_modules/**/*']});
 };
 
+/* Get all js files for test
+ * Return path of all js files
+ */
+let getTestFiles = (path) => {
+    return glob.sync("**/*.spec.js", {'ignore': ['node_modules/**/*']});
```

Comment:
added multiple pattern check, using glob-array. though it's a old module but pretty hefty downloads everyday. 

now this will work for *.spec and *.test almost 77% of the cases :)